Soften language in Invoice and Billing Period Handling appendix entry to avoid expectations (#2101)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/appendix/invoice_and_billing_period_handling.md

At the conclusion of this process, a key objective is for the aggregated [Billed Costs](#datasets.invoicedetail.billedcost) in the Invoice Detail dataset for a given [Invoice Detail ID](#datasets.invoicedetail.invoicedetailid) to align with the payable amounts presented on the corresponding invoice line items. This alignment is performed across [Invoice ID](#datasets.invoicedetail.invoiceid), Invoice Detail ID, and [Invoice Issuer](#datasets.invoicedetail.invoiceissuername).

Similarly, practitioners rely on the aggregated Billed Costs in the Invoice Detail dataset matching the aggregated [Billed Costs](#datasets.costandusage.billedcost) in the Cost and Usage dataset for the same identifiers.

Practitioners may perform *invoice reconciliation* independently by verifying that invoice line items align with data delivered in [*FOCUS dataset artifacts*](#glossary:dataset-artifact), particularly Cost and Usage, Invoice Detail, and Billing Period.

> Once an invoice is issued, it becomes an authoritative financial document, and the information it contains is generally treated as a static historical record.

Reviewer: @shawnalpay In this particular case, I’d suggest keeping the original wording. 

This statement is describing the nature of the **issued invoice itself** (as an authoritative financial document), rather than setting expectations on the underlying data or reconciliation process.

> Once an invoice is issued, it becomes an authoritative financial document, and the information it contains is expected not to change.

Other than this, the PR LGTM

Author: Yes... very good point.  This one is actually grounded in a normative requirement 😅 

Putting it back:

```suggestion
Once an invoice is issued, it becomes an authoritative financial document, and the information it contains is expected not to change.
```
